photon-ml/src/integTest/scala/com/linkedin/photon/ml/avro/ModelProcessingUtilsTest.scala

```diff
@@ -32,6 +32,7 @@ import com.linkedin.photon.ml.supervised.model.GeneralizedLinearModel
 import com.linkedin.photon.ml.TaskType
 import com.linkedin.photon.ml.test.{SparkTestUtils, TestTemplateWithTmpDir}
 import com.linkedin.photon.ml.util._
+import org.apache.spark.SparkContext
```

Comment:
This import block doesn't fit the Spark style guide anymore (which we appropriated for Photon): https://github.com/databricks/scala-style-guide#imports